### PR TITLE
Release 9.17

### PR DESCRIPTION
**Pull request by alex-ju**
## 9.17

* Add Copy to clipboard component (PR #494)
* Bugfix for Radios component (PR #496)
* Fix for Google Tag Manager component (PR #497)
* Support a description for Success Alert component (PR #499)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## Unreleased
+## 9.16.2
 
 * Bugfix for Radios component (PR #496)
```
Comment (tijmenb) [inline CHANGELOG.md]: I think this also includes the copy to clipboard component, sorry forgot to update the changelog.

Comment (alex-ju) [reply CHANGELOG.md]: You're right. Updated to 9.17 and included that PR in the changelog.

File: lib/govuk_publishing_components/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module GovukPublishingComponents
-  VERSION = '9.16.1'.freeze
+  VERSION = '9.17'.freeze
```
Comment (tijmenb) [inline lib/govuk_publishing_components/version.rb]: For the future: this should have been `9.17.0`, as that's [the semver way](https://semver.org/).

Comment (alex-ju) [reply lib/govuk_publishing_components/version.rb]: Yep, forgot the patch. My bad.

**Pull request merged**